### ISSUE TITLE
Keyboard Navigation: Disabled "Go Back" Button in Reference Side Docs Receives Keyboard Focus

### ISSUE THREAD

**Issue by Nina-1210** [microsoft/pxt-microbit#7034]
**Describe the bug**
When navigating via keyboard in the Reference side docs, the disabled "Go Back" button is included in the keyboard tab order and can receive keyboard focus even though it is unavailable and cannot be activated.

**Steps to reproduce the behavior:**
1. Navigate to [Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta).
2. Open an existing project or create a new one.
3. Press **Tab** to focus the **Help** button and press **Enter**.
4. Use the **Down Arrow** key to select **Reference**, then press **Enter** to open the Reference side docs.
5. Press **Tab** until focus reaches the **external link** icon.
6. Press **Tab** again.
7. Observe that focus moves to the disabled "**Go Back**" button.

**Actual behavior**
The disabled "Go Back" button is included in the keyboard tab order and can receive focus even though it is unavailable and cannot be activated.
 
**Expected behavior**
Disabled controls should be removed from the keyboard tab order and should not receive keyboard focus unless there is a specific accessibility reason for them to remain focusable.
 
**Screenshots**
<img width="469" height="605" alt="Image" src="https://github.com/user-attachments/assets/80766ba7-a162-46d9-8cad-8fb7806aa2d6" />

**Additional context**
1.OS: Windows 11 25H2 & Mac Mini M2 macOS Tahoe 26.5.2
2.Browser: Chrome, Edge, Firefox, and Safari
3.makecode.microbit.org version: 9.0.3
4.Microsoft MakeCode version: 13.0.3

**Comment by abchatra** [comment]
@microbit-robert
